src/status_im/ui/components/react.cljs

```diff
@@ -88,17 +88,18 @@
               [text-class (add-font-style :style opts)]
               ts))))))
 
+
 (defn text-input [{:keys [font style] :as opts
                    :or   {font :default}} text]
   (let [font (get-in platform-specific [:fonts (keyword font)])]
     [text-input-class (merge
-                        {:underline-color-android :transparent
-                         :placeholder-text-color  st/text2-color
-                         :placeholder             (i18n/label :t/type-a-message)
-                         :value                   text}
-                        (-> opts
-                            (dissoc :font)
-                            (assoc :style (merge style font))))]))
+                       {:underline-color-android :transparent
```

Comment:
previous indent was ugly but right

Reply:
ok, thanks, will revert this change

Reply:
wait, but why is it wrong? https://github.com/bbatsov/clojure-style-guide#one-space-indent

Reply:
right sorry